internal/pkg/service/buffer/worker/service/conditions_test.go

```diff
@@ -130,22 +130,22 @@ func TestConditionsChecker(t *testing.T) {
 	// Check conditions checker logs
 	wildcards.Assert(t, `
 %A
-[task][slice.swap/%s]INFO  started task "00000123/my-receiver-B/my-export-2/slice.swap/%s"
-[task][slice.swap/%s]DEBUG  lock acquired "runtime/lock/task/00000123/my-receiver-B/my-export-2/slice.swap/%s"
+[task][slice.swap/%s]INFO  started task "00000123/my-receiver-B/slice.swap/%s"
+[task][slice.swap/%s]DEBUG  lock acquired "runtime/lock/task/00000123/my-receiver-B/slice.swap/%s"
 [task][slice.swap/%s]INFO  task succeeded (%s): new slice created, the old is closing
-[task][slice.swap/%s]DEBUG  lock released "runtime/lock/task/00000123/my-receiver-B/my-export-2/slice.swap/%s"
+[task][slice.swap/%s]DEBUG  lock released "runtime/lock/task/00000123/my-receiver-B/slice.swap/%s"
 %A
 `, strhelper.FilterLines(`^(\[task\])`, workerDeps1.DebugLogger().AllMessages()))
 	wildcards.Assert(t, `
```

Comment:
Tady mi to spadne dost záhadně: 
![CleanShot 2023-02-08 at 23 07 48@2x](https://user-images.githubusercontent.com/215660/217662398-c7ce5042-4a44-4ee9-9394-83570535631d.jpg)